voorhoede/head-start
# feature: custom fonts with preloading

# Changes

- Adds custom font import strategy using Fontsource so font files are served from same domain and are automatically hashed for caching (when combined with #207).
- Adds resource hints to preload the fonts for faster page rendering.

Note: based on #208 so the `<PerfHead>` could easily be extended.

# Associated issue

Resolves #87

# How to test

1. Open preview link
2. Verify a custom font is used for the page (body)
3. Open DevTools > Network and verify font files are preloaded

# Checklist

- ~~I have made updated relevant documentation files (in project README, docs/, etc)~~ created follow-up issue #210 
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
